src/Configuration/AutoMapperConfig.php

```diff
@@ -55,12 +55,9 @@ public function getMappingFor(
         string $destinationClassName
     ): ?MappingInterface {
         // Check for an exact match before we try parent classes.
-        foreach ($this->mappings as $mapping) {
-            $isExactMatch = $mapping->getSourceClassName() === $sourceClassName
-                && $mapping->getDestinationClassName() === $destinationClassName;
-            if ($isExactMatch) {
-                return $mapping;
-            }
+        $mapping = @$this->mappings["$sourceClassName|$destinationClassName"];
```

Comment:
`$mapping = $this->mappings["$sourceClassName|$destinationClassName"] ?? null` is faster
```
$test = [];
$start = microtime(true);
for ($i = 0; $i < 10000000; $i++) {
    $result = @$test[$i];
}

echo (microtime(true) - $start) . \PHP_EOL; // 10.039760112762
```
vs
```
$test = [];
$start = microtime(true);
for ($i = 0; $i < 10000000; $i++) {
    $result = $test[$i] ?? null;
}

echo (microtime(true) - $start) . \PHP_EOL; // 2.923015832901
```

Reply:
I was not aware of this operator, thanks.

Reply:
But it was added in PHP 7.0, do I need to support 5.6 and use isset with ternary operator instead?

Reply:
Library did not support it either: https://github.com/mark-gerarts/automapper-plus/blob/master/composer.json#L24

Reply:
This library requires minimal php version as 7.1